trojsten/special/plugin_prask_5_1_1/update_points.py

```diff
@@ -0,0 +1,30 @@
+from trojsten.submit.constants import SUBMIT_TYPE_EXTERNAL
+from trojsten.submit.models import Submit
+from trojsten.contests.models import Task
+
+from .models import UserLevel
+
+ZWARTE_DOOS_TASK_ID = 984
+
+
+def get_task():
+    try:
+        return get_task._cache
+    except AttributeError:
+        get_task._cache = Task.objects.get(pk=ZWARTE_DOOS_TASK_ID)
+        return get_task._cache
+
+
+def update_points(user):
+    points = UserLevel.objects.filter(user=user.id, solved=True).count()
+    submit = Submit(
+        task=get_task(),
+        user=user,
+        points=points,
+        submit_type=SUBMIT_TYPE_EXTERNAL,
+        filepath="",
+        testing_status="OK",
+        tester_response="",
```

Comment:
Prečo tu treba passovať prázdne stringy?

Reply:
I actually don't know, but I am sure it works with them, and that's all that matters.

Reply:
That is exactly why you should know what each of these does and whether they are essential. Otherwise, when someone else looks at the code, he might not know that they are redundant.

Reply:
Ok, they seem redundant. I've removed them.